ModelBundle/Repository/ContentRepository.php

```diff
@@ -374,25 +374,40 @@ public function findPaginatedLastVersionByContentTypeAndSite($contentType = null
      * @param int|null            $siteId
      *
      * @return int
+     *
+     * @deprecated will be removed in 1.3.0, use countByContentTypeAndSiteInLastVersionWithFilter
      */
     public function countByContentTypeInLastVersionWithFilter($contentType = null, FinderConfiguration $configuration = null, $siteId = null)
```

Comment:
can you add a trigger error in deprecated method for instance
`
 @trigger_error('The '.__METHOD__.' method is deprecated since version 1.2.0 and will be removed in 1.3.0', E_USER_DEPRECATED);`